File: VB/SpecifyTextMarkupAnnotationProperties/Form1.vb
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraPdfViewer

Namespace SpecifyTextMarkupAnnotationProperties
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()

            ' Load a document.
            pdfViewer1.LoadDocument("..\..\Demo.pdf")

            ' Handle the TextMarkupAnnotationCreating event to specify the text markup annotation properties when the annotation is being created. 
            AddHandler pdfViewer1.TextMarkupAnnotationCreating, AddressOf pdfViewer1_TextMarkupAnnotationCreating
        End Sub

        Private Sub pdfViewer1_TextMarkupAnnotationCreating(ByVal sender As Object, ByVal e As PdfTextMarkupAnnotationCreatingEventArgs) Handles pdfViewer1.TextMarkupAnnotationCreating
            e.Author = "John Smith"
            e.Comment = "Note."
            e.Color = Color.Gold
        End Sub
    End Class
End Namespace

File: CS/SpecifyTextMarkupAnnotationProperties/Form1.cs
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraPdfViewer;

namespace SpecifyTextMarkupAnnotationProperties {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();

            // Load a document.
            pdfViewer1.LoadDocument("..\\..\\Demo.pdf");

            // Handle the TextMarkupAnnotationCreating event to specify the text markup annotation properties when the annotation is being created. 
            pdfViewer1.TextMarkupAnnotationCreating += pdfViewer1_TextMarkupAnnotationCreating;
        }

        private void pdfViewer1_TextMarkupAnnotationCreating(object sender, PdfTextMarkupAnnotationCreatingEventArgs e) {
            e.Author = "John Smith";
            e.Comment = "Note.";
            e.Color = Color.Gold;
        }
    }
}
